winterfell/test/testClaimController.js

```diff
@@ -210,9 +211,40 @@ describe('SaveClaimController', function () {
       .expect(201, function() {
         Form.findOne({key: '1', user: targetUser._id}, function(error, doc) {
           should.not.exist(error);
+          should.exist(doc);
           doc.responses.should.deepEqual({key1: 'value1', key2: 'value2'});
           done();
         })
       });
   });
+
+  it('Should correctly calculate progress after save', function(done) {
```

Comment:
It's ok to leave this test here for now.
Some somewhere down the line, if we want to test individual forms (like VBA-21-0966 here), we should have one separate test file for testing individual forms. `testClaimController` should just test your general basic claim functions